clang/lib/Index/USRGeneration.cpp

```diff
@@ -1070,6 +1063,24 @@ void USRGenerator::VisitMSGuidDecl(const MSGuidDecl *D) {
   D->NamedDecl::printName(Out);
 }
 
+void USRGenerator::VisitBaseUsingDecl(const BaseUsingDecl *D) {
+  // Add the filename when needed to disambiguate using decls from different
+  // files.
```

Comment:
that much is clear from the code, it would be more useful to say why

our ideas about when using decls are the same vs different are dissimilar to other decls